src/vs/workbench/services/languageRuntime/common/positronDataExplorerComm.ts

```diff
@@ -1429,7 +1430,7 @@ export class PositronDataExplorerComm extends PositronBaseComm {
 	 *
 	 * @returns undefined
 	 */
-	openDataset(uri: string): Promise<OpenDatasetResult> {
+	openDataset(uri: URI): Promise<OpenDatasetResult> {
 		return super.performRpc('open_dataset', ['uri'], [uri]);
 	}
```

Comment:
This code is generated so as I mentioned above we'll have to figure out an alternate approach that is compatible with the code generator. Shouldn't be too hard.